src/Okta.Sdk/Generated/UsersClient.Generated.cs

```diff
@@ -98,6 +98,10 @@ public async Task<IUser> UpdateUserAsync(IUser user, string userId, Cancellation
                 {
                     ["userId"] = userId,
                 },
+                QueryParameters = new Dictionary<string, object>()
+                {
+                    ["strict"] = strict,
```

Comment:
I haven't seen this syntax before.  Aren't Dictionary literal elements usually in the form `{"strict", strict }`?

Reply:
not sure if I follow, what do you mean?